schemas/server.yml

```diff
@@ -59,6 +59,20 @@
       description: >
         Server domain.
 
+    - name: registered_domain
+      level: extended
+      type: keyword
+      short: The highest registered server domain, stripped of the subdomain.
+      description: >
+        The highest registered server domain, stripped of the subdomain.
+
+        For example, the registered domain for "foo.contoso.com" is "contoso.com".
```

Comment:
change plz

Reply:
Jeez, the comments aren't in alphabetical order 🤦‍♂ 

This is a follow-up to [this comment](https://github.com/elastic/ecs/pull/533#discussion_r319588962). Same for all of the other "change plz" comments. Sorry about the curt messages :-)